models/gday/inst/template.job

```diff
@@ -32,9 +32,9 @@ if [ ! -e "@OUTDIR@/.dat" ]; then
   fi
 
   # convert to MsTMIP. Uncomment this code once model2netcdf is written
-  echo "require (PEcAn.GDAY)
-model2netcdf.GDAY('@OUTDIR@', @SITE_LAT@, @SITE_LON@, '@START_DATE@', '@END_DATE@')
-" | R --vanilla
+  #echo "require (PEcAn.GDAY)
```

Comment:
I took this out of the pull request. If you look at the file that are being changed the gday ones are not there.